### PR TITLE
Bug Fixed issue number #195

### PR DESCRIPTION
**Pull request by vishnubansal001**
As the issue raised by me that the section below the navbar which is saying in the marquee type tag that join us etc etc is not showing when we are scrolling down and when we scroll up it is showing so this is being resolved in this pull request

### CODE REVIEW

**Comment by PiyushKalyanpy** [comment]
Please add issue number in description of PR

File: pages/index.tsx [hunk 1]
```diff
@@ -24,19 +24,19 @@ export default function Home() {
   const router = useRouter();
   const [showNav, setShowNav] = useState(true);
 
-  useEffect(() => {
-    let prevScrollpos = window.pageYOffset;
+  // useEffect(() => {
```
Comment (lalitmee) [inline pages/index.tsx:27]: Why are we commenting out this code?

File: package-lock.json [hunk 2]
```diff
@@ -26,15 +24,18 @@
         "react-firebase-hooks": "^5.1.1",
         "react-icons": "^4.8.0",
         "react-redux": "^8.0.5",
+        "react-toastify": "^9.1.3",
```
Comment (lalitmee) [inline package-lock.json:27]: Don't push `package-lock.json` file if you haven't installed any new package in `package.json`

**Review by lalitmee — CHANGES_REQUESTED** [review]

**Comment by lalitmee** [comment]
> As the issue raised by me that the section below the navbar which is saying in the marquee type tag that join us etc etc is not showing when we are scrolling down and when we scroll up it is showing so this is being resolved in this pull request

@ vishnubansal001, that is the desired behavior right now. If someone scrolls down it means they want to see more so we can hide the navbar and as they scroll up we can show the navbar again.

@PiyushKalyanpy, please confirm this and close this PR, if this behavior is desired.

**Comment by PiyushKalyanpy** [comment]
Hey @vishnubansal001 , we have designed it like this only. So that the user can view the full page view by removing obstruction (marquee element : in this case)

**Pull request closed without merging**